apps/condo/pages/tour/guide.tsx

```diff
@@ -188,21 +187,21 @@ const AboutAppBlock = () => {
                                     }
                                 </Col>
                                 {
-                                    get(guideModalCardLink, [locale, type]) && (
+                                    get(guideModalCardReviews, [locale, 'types', type]) && (
                                         <Col xs={24} md={12}>
                                             <Card title={(
                                                 <img
-                                                    src={`/onboarding/guide/aboutApp/${locale}/${type}/modalLogo.webp`}
+                                                    src={get(guideModalCardReviews, [locale, 'types', type, 'imageUrl'], {})}
                                                     style={CARD_IMAGE_STYLES}
                                                 />
                                             )}
                                             >
                                                 <Card.CardBody
-                                                    description={intl.formatMessage({ id: `tour.guide.aboutApp.${type}.modal.card.body` as FormatjsIntl.Message['ids'] })}
+                                                    description={get(guideModalCardReviews, [locale, 'types', type, 'text'])}
                                                     mainLink={{
-                                                        href: get(guideModalCardLink, [locale, type]),
+                                                        href: get(guideModalCardReviews, [locale, 'types', type, 'blogUrl']),
                                                         PreIcon: ExternalLink,
-                                                        label: ModalCardLinkMessage,
+                                                        label: get(guideModalCardReviews, [locale, 'textLink']),
```

Comment:
Looks like this should be a link text :)
```suggestion
                                                        label: get(guideModalCardReviews, [locale, 'linkText']),
```

Reply:
Should this translation be added to the envs?
It seems it will always be the same no matter what reviews you add